plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/width-options-control.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore - Ignoring because `__experimentalToggleGroupControl` is not yet in the type definitions.
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore - Ignoring because `__experimentalToggleGroupControlOption` is not yet in the type definitions.
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+	// @ts-expect-error Using experimental features
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToolsPanelItem as ToolsPanelItem,
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore - Ignoring because `__experimentalUnitControl` is not yet in the type definitions.
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalUnitControl as UnitControl,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { DimensionsControlProps, WidthOptions } from '../../types';
+
+const getHelpText = ( type: WidthOptions ) => {
+	if ( type === WidthOptions.FILL ) {
+		return __( 'Stretch to fill available space.', 'woocommerce' );
+	}
+
+	return __( 'Specify a fixed width.', 'woocommerce' );
+};
+
+const WidthOptionsControl = ( {
+	dimensions,
+	setAttributes,
+}: DimensionsControlProps ) => {
+	const { widthType, fixedWidth = '' } = dimensions;
+	const setDimensions = ( type: WidthOptions ) => {
+		setAttributes( {
+			dimensions: {
+				...dimensions,
+				widthType: type,
+			},
+		} );
+	};
+
+	return (
+		<ToolsPanelItem
+			label={ __( 'Width', 'woocommerce' ) }
+			hasValue={ () => true }
```

Comment:
I suggest using actual conditions for `hasValue` so basically whenever the value is different from a default:

```suggestion
			hasValue={ () => widthType !== WidthOptions.FILL }
			isShownByDefault
```

You can even import the default value from `plugins/woocommerce-blocks/assets/js/blocks/product-collection/constants.ts` for comparison.

If you wanted to make sure option is shown by default, use `isShownByDefault` prop as in the suggestion above.